ports/esp32/boards/deploy_c3.md

```diff
@@ -0,0 +1,14 @@
+Program your board using the esptool.py program, found [here](https://github.com/espressif/esptool).
+
+If you are putting MicroPython on your board for the first time then you should
+first erase the entire flash using:
+
+```bash
+esptool.py --chip esp32 --port /dev/ttyUSB0 erase_flash
+```
+
+From then on program the firmware starting at address 0x1000:
```

Comment:
still needs changing to 0x0000...?

actually, since it's zero, it can probably just be `0` here and in the command below

Reply:
funny how I just couldn't see that.
yes, I'll change it to `0x0`
thanks, mate :)